comet/services/orchestration.py

```
@@ -196,15 +196,16 @@ async def _fetch_cached_rows(self, media_id: str):
 
     async def get_cached_torrents(self):
         rows = []
+        primary_info_hashes = set()
         cache_row_groups = await asyncio.gather(
             *(
                 self._fetch_cached_rows(cache_media_id)
                 for cache_media_id in self.cache_media_ids
             )
         )
         for cache_media_id, cache_rows in zip(self.cache_media_ids, cache_row_groups):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)